examples/reference/ambientLight/ambientLight0/.test.yml

```diff
@@ -0,0 +1,2 @@
+test:
+  reference: https://processing.org/reference/images/ambientLight_0.png
```

Comment:
I notice that the original Processing reference example images are zero-indexed, so the reference directory names and .rpde files are zero-indexed. I believe that the first batch of examples were 1-indexed, like:

> examples/reference/bezierTangent/bezierTangent1/bezierTangent1.rpde
>
> examples/reference/bezierTangent/bezierTangent2/bezierTangent2.rpde

Should all these old files / paths be renamed to match the pattern?